finality-aleph/src/network/manager/discovery.rs

```diff
@@ -92,7 +92,7 @@ impl Discovery {
 
     fn should_broadcast(missing_authorities_num: usize, total_node_count: NodeCount) -> bool {
         // If we are not sure we know of at least one honest node.
```

Comment:
Pls add explanation to the formula used here (with assumptions etc) so we will not change it by accident later

Reply:
Added long comment and made the formula actually BFT.